.yarn/patches/babel-plugin-tsconfig-paths-npm-1.0.3-rc.6-918b52a0d2.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/lib/resolve.js b/lib/resolve.js
+index edc56844e6b8c9b0e2a5b15066a539c8f5a81752..d5df0c9ef052d36be95d2a62c74bc53d94bfb7f7 100644
+--- a/lib/resolve.js
++++ b/lib/resolve.js
+@@ -41,6 +41,7 @@ const getImportTransformer = objKeyCache((aliases, opts) => stringKeyCache((impo
+     try {
+       const stat = fs.statSync(transformedImport)
+       if (stat && stat.isDirectory()) {
++        checkImports.push(path.join(transformedImport, 'index.browser'))
```

Comment:
Why do we need to go that far? Is babel+tsconfig work in the ecosystem not supporting our usecase? If that's the case, maybe let's look into doing something less unexpected. Or fix it upstream if it's a bug.

Reply:
This patch is not needed anymore.